src/Corax/Utils/SpatialHelper.cs

```diff
@@ -0,0 +1,280 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using Sparrow.Server;
+using Spatial4n.Core.Context;
+using Spatial4n.Core.Distance;
+using Spatial4n.Core.Shapes;
+using Spatial4n.Core.Util;
+using Voron;
+using Voron.Data.CompactTrees;
+
+namespace Corax.Utils;
+
+public class SpatialHelper
+{
+    //Source: https://github.com/apache/lucenenet/blob/master/src/Lucene.Net.Spatial/Query/SpatialArgs.cs
+    private static readonly char[] alphabet =
+    {
+        '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'j', 'k', 'm', 'n', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'
+    };
+
```

Comment:
Use a `ReadOnlySpan<byte>` with a `=>` operator to ensure that gets written in the binary and the access is just a memory access.